build.sbt

```diff
@@ -2,65 +2,143 @@ import sbt.ScriptedPlugin
 import sbt.ScriptedPlugin._
 import Dependencies._
 
-version.in(ThisBuild) ~= { old: String =>
```

Comment:
This line should stay, it's useful to testing the sbt plugin and patching broken releases